src/convertToObject.js

```diff
@@ -6,7 +6,19 @@
  * @return {object}
  */
 function convertToObject(sourceString) {
-  // write your code here
+  const cssProperties = {};
+  const arrayOfItems = sourceString
+    .split(';') // ['ключ : свойство','ключ:свойство']
+    .filter((item) => item.includes(':')) // убираем  пустые элементы
+    .map((elem) => elem.split(':')); //[[ключ, свойство], [ключ, свойство]]
```

Comment:
According to the [CODE STYLE] checklist item, all chained methods should have the same indentation to improve readability. Consider formatting the chain of method calls with proper indentation.